casanovo/config.py

```
@@ -29,183 +29,158 @@
-                " devices due to incompatibility with MPS accelerators.\n"
-                "Note: If you want to use a different accelerator (other than MPS),"
-                " please specify it explicitly in the config file."
+                "accelerator='auto' will be overwritten to 'cpu' on Apple "
+                "Silicon devices due to incompatibility with MPS "
+                "accelerators.\nNote: If you want to use a different "
+                "accelerator (other than MPS), please specify it explicitly "
+                "in the config file."
             )
 
-    def __getitem__(self, param: str) -> Union[int, bool, str, Tuple, Dict]:
+    def __getitem__(self, param: str) -> Any:
```

Ruff (0.15.7)

[warning] Dynamically typed expressions (typing.Any) are disallowed in `__getitem__`

(ANN401)